cve_bin_tool/NVDAutoUpdate.py

```diff
@@ -283,3 +283,63 @@ def get_cves(self, *vendor_product_pairs):
     def get_cvelist_if_stale(self):
         """ Update CVEs data from NVD if stale."""
         get_cvelist_if_stale(self.nvddir, self.dbname)
+
+class MappingDb:
```

Comment:
I think "mappingdb" is too generic a name given how many things in this application are "mapped" -- VersionSignatureDb, maybe?